sync_first/sync_first_app/templates/all_incidents_by_id.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container">
+    <div class="jumbotron">
+        <h1 class="display-1 text-center">תוצאות</h1>
+    </div>
+    <h1 class="text-center m-5"><span id="id">{{incident_id}}</span> תוצאות עבור תעודת זהות מספר</h1>

```

Comment:
you sure this is incident id and not person id?

Reply:
wow! great catch